specs/src/tx_client_v2.md

```diff
@@ -0,0 +1,207 @@
+# Tx Client v2 – Sequential Submission Queue
+
+## Abstract
```

Comment:
It's probably too obvious to us all, but I think it should be explicitly stated that this issue is related to submission from single account.